From 8e7d88c035506118a614c3acc0f9073f7cf05754 Mon Sep 17 00:00:00 2001
From: Jiyu Yang <jiyu.yang@amlogic.com>
Date: Tue, 24 Oct 2017 11:29:17 +0800
Subject: [PATCH 12/16] Revert "osd: fixed smem_start error [3/3]"

This cause Android can't work, revert it firstly
This reverts commit 474cffeae2863caa8a0395d55b976d579621afcd.

Change-Id: I6b3b86a6a88fe734900bd9e688bd0683a3008a43
Signed-off-by: Jiyu Yang <jiyu.yang@amlogic.com>
---
 drivers/amlogic/media/osd/osd_fb.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/amlogic/media/osd/osd_fb.c b/drivers/amlogic/media/osd/osd_fb.c
index 35e171f8c4fb..10a85994c0b2 100644
--- a/drivers/amlogic/media/osd/osd_fb.c
+++ b/drivers/amlogic/media/osd/osd_fb.c
@@ -1054,8 +1054,8 @@ static int osd_open(struct fb_info *info, int arg)
 	pr_info("%s, %d, fb_index=%d,fb_rmem_size=%ld\n",
 		__func__, __LINE__, fb_index, fb_rmem_size[fb_index]);
 
-	//fix->smem_start = 0;
-	//fix->smem_len = fb_rmem_size[fb_index];
+	fix->smem_start = 0;
+	fix->smem_len = fb_rmem_size[fb_index];
 	if (!fb_ion_client)
 		fb_ion_client = meson_ion_client_create(-1, "meson-fb");
 	return 0;
-- 
2.11.0

